lib/rack/honeycomb/middleware.rb

```diff
@@ -194,21 +198,34 @@ def add_response_fields(event, status, headers, body)
       end
 
       def with_tracing_if_available(event, env)
```

Comment:
All of the below changes are to support the different versions of the beeline code so that we can follow the rollout plan of releasing the different ruby 'plugin' gems and then release the beeline which contains the changes we are using in here. This situation would be a good reason for folding the gem into one like the other beelines